modules/serverless-rn-1-23-0.adoc

```diff
@@ -0,0 +1,67 @@
+
+{ServerlessProductName} 1.23.0 is now available. New features, changes, and known issues that pertain to {ServerlessProductName} on {product-title} are included in this topic.
+
+[id="new-features-1-23-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.2.
+* {ServerlessProductName} now uses Knative Eventing 1.2.
+* {ServerlessProductName} now uses Kourier 1.2.
+* {ServerlessProductName} now uses Knative `kn` CLI 1.2.
+* {ServerlessProductName} now uses Knative Kafka 1.2.
+* The `kn func` CLI plug-in now uses `func` 0.24.
+
+* It is now possible to use the `kafka.eventing.knative.dev/external.topic` annotation with the Kafka broker. This annotation makes it possible to use an existing externally managed topic instead of the broker creating its own internal topic.
+
+* The `kafka-ch-controller` and `kafka-webhook` Kafka components no longer exist. These components have been replaced by the `kafka-webhook-eventing` component.
+
+* The {FunctionsProductName} Technology Preview now uses Source-to-Image (S2I) by default to build container images.
+
+////
+not identified yet
+
+[id="fixed-issues-1-23-0_{context}"]
+== Fixed issues
+////
+
+[id="known-issues-1-23-0_{context}"]
+== Known issues
+
+* The Federal Information Processing Standards (FIPS) mode is disabled for Kafka broker, Kafka source, and Kafka sink.
+
+* If you delete a namespace that includes a Kafka broker, the namespace finalizer may fail to be removed if the broker's `auth.secret.ref.name` secret is deleted before the broker.
+
+* Running {ServerlessProductName} with a large number of Knative services can cause Knative activator pods to run close to their default memory limits of 600MB. These pods might be restarted if memory consumption reaches this limit. Requests and limits for the activator deployment can be configured by modifying the `KnativeServing` custom resource:
++
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+  namespace: knative-serving
+spec:
+  deployments:
+  - name: activator
+    resources:
+    - container: activator
+      requests:
+        cpu: 300m
+        memory: 60Mi
+      limits:
+        cpu: 1000m
+        memory: 1000Mi
+----
+
+* If you are using link:https://buildpacks.io/[Cloud Native Buildpacks] as the local build strategy for a function, `kn func` is unable to automatically start podman or use an SSH tunnel to a remote daemon. The workaround for these issues is to have a Docker or podman daemon already running on the local development computer before deploying a function.
+
+* On-cluster function builds currently fail for Quarkus and Golang runtimes. They work correctly for Node, Typescript, Python, and Springboot runtimes.
+
+* There is currently a known issue when using a Kafka channel-backed, multi-tenant broker under conditions where the throughput of the Kafka channel outpaces the limit of the container as set by kubernetes.
```

Comment:
I'm curious, which issue is this one in JIRA? Thanks
Otherwise LGTM

Reply:
TBH I can't remember; I think part of the issue is folks adding and removing issues from the links in the release notes PR, it's very hard to keep track.
I think it was from @pierDipi maybe, so he might remember?

Reply:
If we can't identify which Jira this was for I'll remove it.

Reply:
Asked in Slack https://coreos.slack.com/archives/CD87JDUB0/p1655911061891679

Reply:
Cool. Thanks! So let's remove it.